frontend/libs/studio-components/src/components/StudioCodelistEditor/types/CodeListItem.ts

```diff
@@ -0,0 +1,5 @@
+export type CodeListItem = {
+  description?: string;
+  label?: string;
```

Comment:
I'm not sure `label` should be optional. We can discuss this, my thoughts are that the point of a codelist is to have a "data value" and a "display value" for each item.

Reply:
Yes, I'm not sure why I made it optional. It might just have been a blunder from my part. Good catch!